PixelCoord[] left_lane_points
PixelCoord[] right_lane_points

float32 left_gradient
float32 right_gradient
